languages/common-lisp/exercises/concept/basics/.docs/hints.md

```diff
@@ -0,0 +1,20 @@
+## 1. Refresh Lenny on symbols
```

Comment:
The hints in this file are really nice, but they do give away the solution directly. Instead, we try to link to external resources when possible (see [this example](https://github.com/exercism/v3/blob/master/languages/csharp/exercises/concept/strings/.docs/hints.md)). The reason we don't want to explain the solution directly and instead refer to external documentation, is that we then introduce the student to the fact that there is external documentation and that becomes a starting point for the student to know where to look for when wanting to try and see what other methods there are.

Makes sense?

Reply:
The hints file looks great!